tee-apps/sealed-bid-auction/src/api/types.ts

```diff
@@ -34,10 +32,11 @@ export interface AuctionQuote {
 }
 
 export interface AuctionResult {
+    type: string;
+    orderId: string;
     settlementReceiverAddress: string;
     amountOut: bigint;
-    orderId: string;
-    orderData: OrderData;
+    signature: string;
```

Comment:
Could it be relevant to already pass the value solver will have to pass in `fillerData` field already formatted ? --> abi.encode(uint256 amountOut, address settlementReceiver, bytes authorization)

Or do we consider that solvers will want to validates values first and build the append the data themselves ?